### ISSUE TITLE
Readme question

### ISSUE THREAD

**Issue by matanox** [EliteAndroidApps/WhatsApp-GD-Extractor#11]
Does this project extract the raw text messages, or does it just retrieve the encrypted proprietary file format that the whatsapp export uses? thanks in advance!!

**Comment by blackhat235** [comment]
yes it extracts database along with all media files but to read chat extract you need the key
